content/docs/react-without-es6.md

```diff
@@ -100,7 +99,7 @@ class SayHello extends React.Component {
   }
 
   render() {
-    // Because `this.handleClick` is bound, we can use it as an event handler.
+    // Оскільки `this.handleClick` є прив'язаний в конструкторі екземпляра, то ми можемо його використати як обробник подій.
     return (
       <button onClick={this.handleClick}>
         Say hello
```

Comment:
Please translate `Say hello`.

Reply:
Fixed in 8dc67afc134bd5be26a921ffd4a66cc8f1418ecf